rakefile.rb

```diff
@@ -1,197 +1,14 @@
-require 'open-uri'
-require 'openssl'
-
-require_relative 'scripts/copy-dependencies'
 require_relative 'scripts/utils'
 require_relative 'scripts/R'
-require_relative 'scripts/colorize'
-
-OpenSSL::SSL::VERIFY_PEER = OpenSSL::SSL::VERIFY_NONE
-
-APPVEYOR_ACCOUNT_NAME = 'open-systems-pharmacology-ci'
 
 task :prepare_for_build, [:build_version] do |t, args|
   update_package_version(args.build_version, description_file)
```

Comment:
That should be the only thing required. it will update the package version 2.0.0 to be 2.0.BUILD NUMBER